### PR TITLE
[Torch] Support hard_swish op

### PR DESCRIPTION
**Pull request by Xuxue1**
Closes https://github.com/apache/tvm/issues/6897

Support  hard_swish op https://pytorch.org/docs/stable/generated/torch.nn.Hardswish.html#torch.nn.Hardswish
please review @masahi

### CODE REVIEW

**Comment by masahi** [comment]
@Xuxue1 Great! please run `black pytorch.py`

cc @zhiqwang

File: tests/python/frontend/pytorch/test_forward.py [hunk 1]
```diff
@@ -3437,6 +3437,16 @@ def test_fn(x, weights=None):
     verify_trace_model(test_fn, [inp, weights], targets)
 
 
+def test_hard_swish():
+    hard_swish = torch.nn.Hardswish()
+    hard_swish_inplace = torch.nn.Hardswish(inplace=True)
+    examples = [torch.rand(8), torch.rand(8, 8), torch.rand(8, 16), torch.rand(1, 1, 8)]
+    targets = ["llvm", "cuda"]
+    for input in examples:
+        verify_trace_model(hard_swish, [input], targets)
```
Comment (masahi) [inline tests/python/frontend/pytorch/test_forward.py:3446]: `verify_trace_model` is supposed to be used for dynamic models that require VM compilation. Please use `verify_model` instead.

Comment (Xuxue1) [reply tests/python/frontend/pytorch/test_forward.py:3446]: ok

File: tests/python/frontend/pytorch/test_forward.py [hunk 2]
```diff
@@ -3437,6 +3437,12 @@ def test_fn(x, weights=None):
     verify_trace_model(test_fn, [inp, weights], targets)
 
 
+def test_hard_swish():
+    examples = [torch.rand(8).float(), torch.rand(8, 10).float(), torch.rand(1, 1, 10).float()]
+    for input in examples:
+        verify_model(torch.nn.Hardswish().eval(), input_data=input)
```
Comment (insop) [inline tests/python/frontend/pytorch/test_forward.py:3443]: Any reason to drop `hard_swish_inplace`?

Comment (Xuxue1) [reply tests/python/frontend/pytorch/test_forward.py:3443]: hard_swish_inplace is not a pure function. It will change  input tensor

Comment (Xuxue1) [reply tests/python/frontend/pytorch/test_forward.py:3443]: I can modify verify_model function

Comment (insop) [reply tests/python/frontend/pytorch/test_forward.py:3443]: Looks great!

**Review by masahi — APPROVED** [review]

**Comment by masahi** [comment]
Thanks @Xuxue1

**Pull request merged**